pkgs/os-specific/linux/fscryptctl/legacy.nix

```diff
@@ -34,7 +34,7 @@ stdenv.mkDerivation rec {
       As fscryptctl is intended for advanced users, you should read the kernel
       documentation for filesystem encryption before using fscryptctl.
     '';
-    inherit (src.meta) homepage;
+    inherit (src.meta) homepage changelog;
```

Comment:
"https://github.com/google/fscryptctl/releases/tag/v{version}"

Broken URL.

Reply:
Pebcak? https://github.com/google/fscryptctl/releases/tag/v0.1.0

Reply:
wait no, wtf is happening here

Reply:
Also this package should be dropped accoding to the comment?

Reply:
Oops, right, that package (`fscryptctl-experimental`) should be dropped (I forgot to include my GH handle in the TODO which I use to find them)  and it seems like I forgot the `$` in the `fscryptctl` package. I've opened #144885 if that helps. You can also include it in this PR if you want.